Database based Diagram

Khaled-AlKhateeb opened the pull request:
##Tasks Done: ✔️ 

- created diagram for vet_clinic database using drawio.net.
- added the diagram to the repo. 

![Screenshot 2022-12-20 at 14 20 25](https://user-images.githubusercontent.com/98466955/208703780-d8b37737-a6d3-4cdf-ac0e-73031db5a9c6.png)


![Screenshot 2022-12-20 at 14 25 12](https://user-images.githubusercontent.com/98466955/208703800-881f92f6-74a2-449e-a649-0cbcf6ddb85e.png)

Khaled-AlKhateeb committed:
add database diagram

Munsa1 reviewed (changes requested):
Hi @Khaled-AlKhateeb, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted the wrong project for this milestone

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

jaamanu committed:
Create schema_based_on_diagram.sql

Create a file named schema_based_on_diagram.sql.
Implement the database from the diagram.
Join tables from many-to-many relationships
 Add the FK indexes

sja-thedude reviewed (approved):
## Project Approved 👍 

Hi @Khaled-AlKhateeb & @jaamanu, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All requirements completed
- Professional readme

![giphy](https://user-images.githubusercontent.com/70269507/159485323-11dfc0c5-bd53-4956-b55a-2fc42f7a3845.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
